shelley/chain-and-ledger/formal-spec/crypto-primitives.tex

```diff
@@ -200,4 +203,11 @@ \section{Cryptographic primitives}
   \label{fig:types-msig}
 \end{figure*}
 
+(NSF: define $\type{RequireAnyOf}$ as an abbreviation of $\type{RequireMOf}~1$?
+Or maybe just point out the redundancy?)
```

Comment:
I agree that the redundancy seem weird, I like the idea of removing `RequireAnyOf`, but let me ask around first to make sure I'm not missing something.